mandatory/3-function-output.js

```diff
@@ -1,16 +1,21 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// method to create a function that will return a random integer between 0 and 9 values 
```

Comment:
Not only returns the integer but any number between 0 and 10, that number can sometimes be 9.99